src/common/ExportSettingsModal/ExportSettingsModalContainer.js

```diff
@@ -45,11 +46,14 @@ const ExportSettingsModalContainer = ({
 
       onCancel();
 
+      const filename = `order-export-${moment().format('YYYY-MM-DD-hh:mm')}`;
```

Comment:
IMHO, we already have `intl` as a dependency here and no need to include one more on `moment`. 
`intl.formatDate()` can do the trick

Reply:
`intl.formatDate()` - returns date in `YYYY-MM-DD` format. And if add `hh:mm` props, it returns file name smth like - `order-export-1_26_2021, 11_20 AM`. In this case I have to remove spaces. Take a look plz

Reply:
K, if you have specific format requirements for any locale, moment's way looks the best